vector-stores/spring-ai-weaviate-store/src/test/java/org/springframework/ai/vectorstore/weaviate/WeaviateVectorStoreIT.java

```diff
@@ -359,6 +361,77 @@ public void addAndSearchWithCustomContentFieldName() {
 		});
 	}
 
+	@ParameterizedTest(name = "{0} : {displayName} ")
+	@ValueSource(strings = { "custom_", "" })
+	public void addAndSearchWithCustomMetaFieldPrefix(String metaFieldPrefix) {
```

Comment:
Could you help me understand where the configurable metadata prefix is used in this test and then assert that it takes effect?

Reply:
@sobychacko 

In this test, the following behavior is expected:

1. Verify that data is stored and retrieved correctly using a `customVectorStore` with a modified prefix.
2. Confirm that data cannot be retrieved using a `vectorStore` with the default prefix (as the prefixes differ, retrieval should fail).
3. Remove the data from the `customVectorStore` to ensure it does not affect subsequent tests.

If there’s a better approach, I’m open to suggestions for improvement. 👍